src/attendee/api/views.py

```diff
@@ -1,9 +1,8 @@
 from django.conf import settings
-from registry.helper import reg
-from rest_framework import views
+from rest_framework import views, status
 from rest_framework.response import Response
-from rest_framework import status
 
+from registry.helper import reg
 from ext2020.models import Attendee
```

Comment:
sort the imports by:

1. native packages
2. 3rd party libraries
3. customized packages

and use an empty line to separate them.

```suggestion
from registry.helper import reg

from ext2020.models import Attendee
```